cmd/start.go

```diff
@@ -267,30 +273,30 @@ func initializeBalanceMonitoring(
 	ctx context.Context,
 	chainProvider chain.Handle,
 	config *config.Config,
-	ethereumAddress string,
+	ethereumKey *keystore.Key,
```

Comment:
Why do we pass the entire `*keystore.Key` instead of passing just the address?

Reply:
Good point, see https://github.com/keep-network/keep-core/commit/b02c0e42d1a8db35f487348cb557145be2de8eab